### PR TITLE
Remove grid from brand

### LINKED ISSUE #827: Remove grid from Brand component
**Is your feature request related to a problem? Please describe.**
We previously implemented a grid for the brand component so that the SVG could be indented to align with the content in the central column. This requirement has been removed by UX and therefore our need for grid in this component is no more. 

**Describe the solution you'd like**
Remove all uses of grid from the brand component.

**Describe alternatives you've considered**
N/A

**Testing notes**
Visual regression of both the header and footer which use the brand component. Snapshots should also no longer contain grid. 

- [x] Initially labelled with ["Refinement needed"](https://github.com/BBC-News/simorgh/labels/Refinement%20Needed)

### PR DESCRIPTION
**Pull request by pjlee11**
Resolves #827

- Removes the grid from the Brand component which is integrated in the Footer and Header components
- Makes use of the `layoutWrapperWithoutGrid` const created in #901 
- Removed the logic to indent the Brand component to match the grid content after UX confirmed this is no longer a valid use case
- Without the indent logic there is a redundant styled.div (what was `StyledGridItem`) this is now removed
- Removed both tests and stories relevant to indenting the logo

- [x] Tests added for new features
- [ ] Test engineer approval

### CODE REVIEW

**Comment by pjlee11** [comment]
Blocked awaiting refinement on the issue and the merging of #901

**Comment by pjlee11** [comment]
Issue has been refined

File: src/app/components/Brand/index.jsx [hunk 1]
```diff
@@ -60,46 +55,30 @@ const BrandSvg = styled.svg`
...
-Brand.defaultProps = {
-  indentedLogo: true,
-};
-
-Brand.propTypes = {
-  indentedLogo: bool,
-};
+const Brand = () => (
+  <StyledWrapper>
+    <StyledLink href="https://www.bbc.co.uk/news">
```
Comment (ChrisBAshton) [inline src/app/components/Brand/index.jsx:60]: Hmmm this will apply to all languages, right? Is that what we want?

**Review by ChrisBAshton — DISMISSED** [review]
Code looks ok (got a question on hardcoded brand url but that's existing code that's been moved, rather than new code that's been added).

Responsibility of this PR will mostly lie in the UX Review. 👍

**Review by jtart — DISMISSED** [review]

File: src/app/components/Brand/index.jsx (hunk 1 above)
Comment (pjlee11) [reply src/app/components/Brand/index.jsx:60]: I think we could raise an issue to address this and put it in the Persian project

Comment (ChrisBAshton) [reply src/app/components/Brand/index.jsx:60]: Raised #938.

**Review by ChrisBAshton — APPROVED** [review]
(Well documented ticket description btw 👍)

**Review by bcmn — APPROVED** [review]

**Comment by bcmn** [comment]
Phil has already gone over these changes with UX.

**Comment by jamesbrumpton** [comment]
LGTM

**Pull request merged**